pimd/pimd.h

```diff
@@ -182,11 +183,12 @@ extern uint8_t qpim_ecmp_rebalance_enable;
 #define PIM_DEBUG_PIM_NHT_RP (router->debugs & PIM_MASK_PIM_NHT_RP)
 #define PIM_DEBUG_MTRACE (router->debugs & PIM_MASK_MTRACE)
 #define PIM_DEBUG_VXLAN (router->debugs & PIM_MASK_VXLAN)
+#define PIM_DEBUG_BSM	(router->debugs & PIM_MASK_BSM_PROC)
 
 #define PIM_DEBUG_EVENTS                                                       \
 	(router->debugs                                                        \
 	 & (PIM_MASK_PIM_EVENTS | PIM_MASK_IGMP_EVENTS                         \
-	    | PIM_MASK_MSDP_EVENTS))
+	    | PIM_MASK_MSDP_EVENTS | PIM_MASK_BSM_PROC))
```

Comment:
let's make a `debug pim bsm` instead of tacking this onto event debugging.

Reply:
We have added debug pim bsm too. This is in addition to that.